testing/regress/ecl/key/kafkatest.xml

```diff
@@ -0,0 +1,44 @@
+<Dataset name='PingToCreateTopic'>
+ <Row><PingToCreateTopic>0</PingToCreateTopic></Row>
+</Dataset>
+<Dataset name='ConsumerGetTopicPartitionCount'>
+ <Row><ConsumerGetTopicPartitionCount>1</ConsumerGetTopicPartitionCount></Row>
+</Dataset>
+<Dataset name='ConsumerResetMessageOffsets1'>
+ <Row><ConsumerResetMessageOffsets1>1</ConsumerResetMessageOffsets1></Row>
+</Dataset>
+<Dataset name='PublisherGetTopicPartitionCount'>
+ <Row><PublisherGetTopicPartitionCount>1</PublisherGetTopicPartitionCount></Row>
+</Dataset>
+<Dataset name='PublishMessageUnkeyed'>
+ <Row><PublishMessageUnkeyed>true</PublishMessageUnkeyed></Row>
+</Dataset>
+<Dataset name='PublishMessageKeyed'>
+ <Row><PublishMessageKeyed>true</PublishMessageKeyed></Row>
+</Dataset>
+<Dataset name='GetMessages1'>
+ <Row><paritionnum>0</paritionnum><offset>0</offset><message>Regular message</message></Row>
+ <Row><paritionnum>0</paritionnum><offset>1</offset><message>Keyed message</message></Row>
+</Dataset>
```

Comment:
Added small delay between publishing and consuming steps in order to give time to background copy process to work.